src/clj/pyregence/authentication.clj

```diff
@@ -21,7 +22,11 @@
                                         :user-name             (:user_name user-data)
                                         :user-role             (:user_role user-data)
                                         :organization-id       (:organization_rid user-data)
-                                        :org-membership-status (:org_membership_status user-data)}
+                                        :org-membership-status (:org_membership_status user-data)
+                                        ;; TODO is there a better way to format this DATE?
```

Comment:
We might have some helper functions that are useful in `src/cljs/pyregence/utils/time_utils.cljs`